site/examples/inputs.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { HtmlExample } from '../html_example';
+
+const inputWithButton = `<fieldset class='relative'>
+  <button class='btn absolute top right w96 round-r'>Search</button>
+  <input class='input pr96' placeholder='Search'>
```

Comment:
hmm? are you seeing this spacing as tight?

<img width="176" alt="screen shot 2017-01-17 at 2 06 52 pm" src="https://cloud.githubusercontent.com/assets/61150/22035509/3f7b9194-dcbe-11e6-9865-25587ed62d4b.png">

I wanted to avoid adding too many additional classes.